### DISCUSSION TITLE
ch3-asm/ch3-02-arch

### DISCUSSION THREAD

**Discussion by giscus** [chai2010/advanced-go-programming-book#637] [General]
# ch3-asm/ch3-02-arch

Go语言高级编程

https://chai2010.cn/advanced-go-programming-book/ch3-asm/ch3-02-arch.html

**Comment by Anthony-Dong** [comment]
“引入了 PC、FP、SP、SB 四个伪寄存器”，汇编还要再抽象一层，哈哈哈
